src/main/java/org/opensearch/ad/AnomalyDetectorPlugin.java

```diff
@@ -42,7 +42,7 @@
 import org.opensearch.ad.cluster.ADDataMigrator;
 import org.opensearch.ad.cluster.ClusterManagerEventListener;
 import org.opensearch.ad.cluster.HashRing;
-import org.opensearch.ad.constant.CommonName;
+import org.opensearch.ad.constant.ADCommonName;
```

Comment:
This makes sense at the moment. But with forecasting added, eventually there will be a question of ADCommonName being appropriate. But that bridge can be crossed once one arrives at it  ....